.github/workflows/translation-advice.yml

```diff
@@ -0,0 +1,37 @@
+name: authentik-translation-advice
+
+on:
+  pull_request:
+    branches:
+      - main
+      - translations-comments
+    paths:
+      - "/locale/**"
+      - "/web/src/locales/**"
+
+jobs:
+  post-comment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Find Comment
+        uses: peter-evans/find-comment@v2
+        id: fc
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          comment-author: "github-actions[bot]"
+          body-includes: authentik translations instructions
+      - name: Create or update comment
+        uses: peter-evans/create-or-update-comment@v2
```

Comment:
We could use the bot accounts PAT to create the comment for consistency

Reply:
Those credentials won't be available on community PRs, will they?

Reply:
oh true I keep forgetting that